start.js

```diff
@@ -0,0 +1,68 @@
+const http = require('http');
+const soldiers = require('./routes/soldiers.js');
+const duties = require('./routes/duties.js');
+const justiceBoard = require('./routes/justiceBoard.js');
+const dutiesUrlLength = 12
+const soldiersUrlLength = 14
+
+
+function checkErr(err) {
```

Comment:
You don't have to check the err.
You are using this method only in places where an error is definitely caused by a bad request.

And in general,
A method that has to keep track of an increasing number of random strings - is a bad idea.
If you see yourself doing things like that - seek help and consult how to write it correctly.